docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,550 @@
+
+## Prerequisites
+
+Because `osnadmin` commands are performed against an ordering service node (which executes the actions), the ordering service needs to exist before you can create a channel. You can attempt this tutorial with your existing ordering service or deploy a new ordering service. If you decide to use `osnadmin` commands with an existing ordering service, the system channel must first be removed from each ordering node before you can create any new channels. Choose whether you want to use your existing ordering service or deploy a new one:
+
+- [Use existing ordering service](#use-existing-ordering-service)
+- [Deploy a new ordering service](#deploy-a-new-ordering-service)
+
+### Use existing ordering service
+
+Before you can take advantage of this feature on a deployed ordering service, you need to remove the system channel from each ordering node that is a consenter in your application channels. A "mixed mode" of orderers on a channel, where some nodes are part of a system channel and others are not, is not supported. The `osadmin` CLI includes both a `channel list` and a `channel remove` command to facilitate the process of removing the system channel. If you prefer to deploy a new ordering service instead, skip ahead to [Deploy a new ordering service](#deploy-a-new-ordering-service).
+
+#### Remove the system channel
+
+- Before attempting these steps ensure that you have [upgraded](../upgrade.html) your Fabric network to v2.3 or higher.
+- Modify the `orderer.yaml` for each ordering node to support this feature and restart the node. See the orderer [sampleconfig](https://github.com/hyperledger/fabric/blob/{BRANCH}/sampleconfig/orderer.yaml) for more information about these required parameters.
+    - `General.BootstrapMethod` - Set this value to `none`. Because the system channel is no longer required, the `orderer.yaml` file on each orderer needs to be configured with `BootstrapMethod: none` which means that no bootstrap block is required or used to start up the ordering service.
+    - `Admin.ListenAddress` - The orderer admin server address (host and port) that can be used by the `osnadmin` command to configure channels on the ordering service. This value should be a unique `host:port` combination to avoid conflicts.
+    - `Admin.TLS.Enabled:` - This value must be set to `true`.
+    - `Admin.TLS.PrivateKey:` - The path to and file name of the orderer private key issued by the TLS CA.
+    - `Admin.TLS.Certificate:` - The path to and file name of the orderer signed certificate issued by the TLS CA.
+    - `Admin.TLS.RootCAs:`  - The path to and file name of the orderer TLS CA Root certificate.
```

Comment:
This variable is not useful, I would not document it (and we can/should remove it from the config sample).